### DISCUSSION TITLE
Documentacion ADempiere 394

### DISCUSSION THREAD

**Discussion by AlbertoAbudinen** [adempiere/adempiere#4122] [General]
El objetivo de esta discusión es generar un listado de manuales, inicialmente en drive para que todos puedan aportar, y luego cuando tengamos las cuentas de ADempiere oficial subir los links a la web y/o github o similar.

El temario de los manuales a realizar serán (esto es un trabajo en transito )  :

1. login de usuario
2. introducción a ADempiere y ventanas
3. Maestros
4. flujo de compra estandar 
5. flujo de venta estándar

vamos a ir respondiendo este misma discusión a medida que se vayan generando los manuales y  videos. y pueden aportar con cualquier link aquí mismo también.

El repositorio de los manuales que se van desarrollando los dejare aquí 

https://drive.google.com/drive/folders/1i45386IlyCGKpYelyfc8lTVBhFd3-yg8?usp=sharing

**Comment by yamelsenih** [comment]
Hola @AlbertoAbudinen creo que es bueno considerar el trabajo que ya se hizo en el wiki y sacar de ahí todo lo que se pueda. Por otra parte ya desde hace creo que un año está disponible la web de ADempiere para comenzar a documentar y no se ha subido nada. 

Hace poco aprobé un pull request de mi equipo para comenzar a migrar parte de la wiki pero considero que es un trabajo que desde hace mucho se podía hacer sin impedimento.

Saludos.

Nota: en lugar de colocar aquí los temarios creo que ya se puede hacer de una vez un pull request conos mismos al proyecto https://github.com/adempiere/adempiere-site

**Reply by AlbertoAbudinen** [reply]
Hola Yamel, vamos a ir complementando los manuales, en base a la wiki y todo el resto que tengamos  y luego lo dejamos en el lugar correcto pero lo primero es ordenar bien y actualizar tanto los manuales como también los videos

Por ahora  este es el link del repositorio 

https://drive.google.com/drive/folders/1i45386IlyCGKpYelyfc8lTVBhFd3-yg8?usp=sharing

**Reply by yamelsenih** [reply]
Hola, nosotros comenzamos a migrar paso a paso la Wiki, voy a colocar aquí el avance si gustan
